docs/content/en/docs/clustermgmt/cluster-backup-restore/restore-cluster.md

```diff
@@ -0,0 +1,148 @@
+    ```
+
+1. Edit the cluster config file of the workload clusters.
+
+    You need to update the cluster config file of all the workload clusters to point to the new management cluster created in step 1. For example,
+
+    workload cluster w01
+
+    ```yaml
+    apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+    kind: Cluster
+    metadata:
+      name: w01
+      namespace: default
+    spec:
+      managementCluster:
+        name: mgmt-new # This needs to be updated with the new management cluster name.
+      ...
+    ```
+
+    workload cluster w02
+
+    ```yaml
+    apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+    kind: Cluster
+    metadata:
+      name: w02
+      namespace: default
+    spec:
+      managementCluster:
+        name: mgmt-new # This needs to be updated with the new management cluster name.
+      ...
+    ```
+
+    Make sure that apart from the `managementCluster` field you updated above, all the other cluster config of the workload clusters need to stay the same as the old workload clusters resources after the old management cluster fails.
+
+1. Apply the updated cluster config of the workload clusters in the new management cluster.
+
+    ```bash
+    MGMT_CLUSTER_NEW="mgmt-new"
+    MGMT_CLUSTER_NEW_KUBECONFIG=${MGMT_CLUSTER_NEW}/${MGMT_CLUSTER_NEW}-eks-a-cluster.kubeconfig
+
+    kubectl apply -f w01/w01-eks-a-cluster.yaml --kubeconfig ${MGMT_CLUSTER_NEW_KUBECONFIG}
+    kubectl apply -f w02/w02-eks-a-cluster.yaml --kubeconfig ${MGMT_CLUSTER_NEW_KUBECONFIG}
+    ```
+
+1. Validate all clusters are in the desired state.
+
+    ```bash
+    kubectl get clusters -n default --kubeconfig ${MGMT_CLUSTER_NEW}/${MGMT_CLUSTER_NEW}-eks-a-cluster.kubeconfig
+
+    NAME       AGE
+    mgmt-new   13h
+    w01        11h
+    w02        11h
+
+    kubectl get clusters.cluster.x-k8s.io -n eksa-system --kubeconfig ${MGMT_CLUSTER_NEW}/${MGMT_CLUSTER_NEW}-eks-a-cluster.kubeconfig
+
+    NAME       PHASE         AGE
+    mgmt-new   Provisioned   11h   
+    w01        Provisioned   11h   
+    w02        Provisioned   11h 
+
+    kubectl get kcp -n eksa-system  --kubeconfig ${MGMT_CLUSTER_NEW}/${MGMT_CLUSTER_NEW}-eks-a-cluster.kubeconfig
+
+    NAME       CLUSTER    INITIALIZED   API SERVER AVAILABLE   REPLICAS   READY   UPDATED   UNAVAILABLE   AGE   VERSION
+    mgmt-new   mgmt-new   true          true                   2          2       2                       11h   v1.27.1-eks-1-27-4
+    w01        w01        true          true                   2          2       2                       11h   v1.27.1-eks-1-27-4
+    w02        w02        true          true                   2          2       2                       11h   v1.27.1-eks-1-27-4
+    ```
+
+## Restore a workload cluster 
+
+Similar to failed management cluster without infrastructure components change, follow the [External etcd backup and restore]({{< relref "../etcd-backup-restore/etcdbackup" >}}) to restore the workload cluster itself from the backup.
```

Comment:
Are you also planning on adding a section here for restoring a workload cluster if infrastructure changes do happen? 
For example: during a workload cluster upgrade, if all the CP got rolled out but the upgrade failed during worker upgrade, a simple ETCD restore won't work, since doing a restore would cause the node names, IPs and potentially other things to revert to a state that is longer valid.
Etcd restore is really only a good idea if users lose only their etcd cluster entirely and want to recover their data, or they want to revert their own deployments to the previous state and nothing else in the infra (nodes specifically) has changed.

Reply:
yea i was kinda avoid talking about this until we have a solution for the case where workload cluster is completely inaccessible / infrastructure changed. But you are right, let me add these notes for clarification.